### ISSUE TITLE
Confusing names of variables

### ISSUE THREAD

**Issue by dariagrudzien** [apiaryio/s3-streaming-upload#66]
```
  accessKey: process.env.AWS_S3_ACCESS_KEY,
  secretKey: process.env.AWS_S3_SECRET_KEY,
  bucket: process.env.AWS_S3_TEST_BUCKET,
```

These should be OCI keys when setting the tools to work with OCI Object Storage. Naming of the environment variables is confusing in that situation.

**Comment by Almad** [comment]
I'd vote for generic name for both—s/AWS_S3/BUCKET

**Comment by abtris** [comment]
I agree with premise. But I think that will broke current use for many people using library.

**Comment by abtris** [comment]
I make change just in readme https://github.com/apiaryio/s3-streaming-upload#example-usage-with-oracle-cloud-oci-compatible-s3-api
